### PR TITLE
Ppolk dods 99

### PR DESCRIPTION
**Pull request by ppolk-wave**

### CODE REVIEW

File: streamlit_dashboards/eeg_visualization.py [hunk 1]
```diff
@@ -130,18 +121,29 @@ def eeg_visualization_dashboard():
...
-                        st.session_state.sensitivity = st.select_slider(
-                            "Sensitivity",
+                        st.session_state.sensitivity = st.selectbox(
+                            "Sensitivity in uV",
                             options=sensitivity_options,
-                            value=st.session_state.sensitivity  # Persisted value
+                            index=6,  # Persisted value
                         )
+                    
+                    with sub_col3:
```
Comment (luis-wave) [inline streamlit_dashboards/eeg_visualization.py:142]: Take this out for now, if you are not going to use them in this release

File: dsp/graph_preprocessing.py [hunk 2]
```diff
@@ -82,7 +105,11 @@ def get_min_max_stats(columns, df):
     # new norm: scale to -1, 1 and then adjust it to a percentage of so clean waveforms
     #   arent reaching the bound (on average)
     bound = (median_max + abs(median_min)) / 2
-    scaled_eeg = (df_eeg / bound) * 0.25 * sensitivity_slider ###
+    # scaled_eeg = (df_eeg / bound) * 0.25 * sensitivity_slider ###
```
Comment (luis-wave) [inline dsp/graph_preprocessing.py:108]: Remove unused code instead of commenting them out.

File: dsp/graph_preprocessing.py [hunk 3]
```diff
@@ -82,7 +105,11 @@ def get_min_max_stats(columns, df):
...
+
+    # DODS-99 -->
```
Comment (luis-wave) [inline dsp/graph_preprocessing.py:110]: Be more descriptive with the comments. Outline why the line is there

File: streamlit_dashboards/eeg_visualization.py [hunk 4]
```diff
@@ -68,7 +68,7 @@ def eeg_visualization_dashboard():
                 if "ref" in query_params:
                     selected_reference = query_params["ref"]
                 with col1:
-                    sub_col1, sub_col2 = st.columns(2)
+                    sub_col1, sub_col2, sub_col3, sub_col4 = st.columns(4)
```
Comment (luis-wave) [inline streamlit_dashboards/eeg_visualization.py:71]: Change back to two columns, unless you are planning additional items. If not, save it for the next release.

**Review by luis-wave — COMMENTED** [review]
Great job in adding more granular control for sensitivity. We can proceed once the comments are addressed.

File: streamlit_dashboards/eeg_visualization.py (hunk 1 above)
Comment (ppolk-wave) [reply streamlit_dashboards/eeg_visualization.py:142]: These are dummy columns used to decrease the width of the montage and sensitivity dropdwon widgets. Its a UI enhancement

Comment (ppolk-wave) [reply streamlit_dashboards/eeg_visualization.py:142]: I added a comment to now explain this

**Review by luis-wave — APPROVED** [review]

**Pull request merged**